tests/integration/test_arithmetic.py

```diff
@@ -57,6 +58,7 @@ def main() -> int:
     run_int_fn(compiled, 6)
 
 
+@pytest.mark.skip(reason="trunc_u has an invalid signature")
```

Comment:
What's going on here?

Reply:
Not quite sure, I think guppy defined an invalid signature.
It's getting fixed in #454, so I didn't investigate much.